### DISCUSSION TITLE
FAQs

### DISCUSSION THREAD

**Discussion by RoboMWM** [GriefPrevention/GriefPrevention#752] [Q&A]
# Moved to https://docs.griefprevention.com/faq/

**Comment by RoboMWM** [comment]
#### Common feature requests

- [Can you enforce a minimum distance between claims?](124)
- [Vertical/3D subclaims](269)
- [Permanent/auto /ic](991)

**Comment by RoboMWM** [comment]
If you would like to add an entry here, please comment with the question and answer below! Please use the following format:
```html
<details>
<summary><b>Question goes here</b></summary>

Answer goes here
</details>
```

**Comment by RoboMWM** [comment]
## Please note that comments here are only for submitting FAQs to the list! See this post for the template: https://github.com/TechFortress/GriefPrevention/issues/752#issuecomment-599151463

(I.e. don't make me lock this issue cuz ppl can't read)

**Comment by jonesman103** [comment]
hey: I have a problem. people on my server can not open doors unless they have op.

**Comment by botta96** [comment]
Hi, I use this plugin for my survival server, think about creating VIP packages with the possibility of creating larger complaints, how can I do? I have tried in several ways but I cannot provide the creation of claims larger than the standard format inserted in the configuration file Thanks

**Comment by RoboMWM** [comment]
- Added entry `/claimexplosions does not persist when the claim owner logs off/server restarts`

**Comment by Robomnz** [comment]
![image](https://user-images.githubusercontent.com/34904608/92544125-37520500-f2a1-11ea-8efd-30bf630477ae.png)
i did /containertrust public on a subdivided admin claim and i could not access any of the chests/barrels/enchantment tables, furnaces or even anvils, im using the latest grief prevention version 16.14.0 and running a paper 1.16.2 server is this a bug?

**Comment by Jikoo** [comment]
That message is not from GriefPrevention. If memory serves, that's from EssentialsProtect.

**Comment by Robomnz** [comment]
ah ok lemme remove that real quick and see if that fixes it

**Comment by Robomnz** [comment]
![image](https://user-images.githubusercontent.com/34904608/92545549-7f265b80-f2a4-11ea-9b1d-c180580f67f6.png)
i removed it and made sure it was set to public for containertrust but its still happening
![image](https://user-images.githubusercontent.com/34904608/92545696-cb719b80-f2a4-11ea-9004-cf1f57b422d7.png)
edit: even accesstrust on the main admin claim wont work 

edit again: those are permissions: access trust on main claim and container trust on subdivided claim also i wasnt able to have both access trust and container trust set to public on a single claim it would set it to one or the other.
![image](https://user-images.githubusercontent.com/34904608/92545781-0378de80-f2a5-11ea-831f-9f1ebc08282e.png)

**Comment by Jikoo** [comment]
Again, those messages are not from GP. Also, if you're having in issue, please open an issue instead of posting here. Please make sure you restarted your server after removing EssentialsProtect.

**Comment by Robomnz** [comment]
i did reboot my server after removing it and the issue is still there

**Comment by RoboMWM** [comment]
Some people really can't be bothered to read anything can they.

# Please note that comments here are only for submitting FAQs to the list!

## (I.e. don't make me lock this issue cuz ppl can't read)

**Comment by SaltyLTS** [comment]
![image](https://user-images.githubusercontent.com/52759884/94318476-f3414e80-ff88-11ea-9c64-44736772d505.png)
Is there a way to put the number of claim blocs we have in the tab ? i use the TAB plugin

**Comment by RoboMWM** [comment]
Gonna lock this since nobody has contributed to the FAQ and I'm making a documentation website repo elsewhere that should replace this.
